plugins/indexer/plugin.go

```diff
@@ -102,31 +107,31 @@ func configure() {
 		Plugin.LogPanicf("failed to start worker: %s", err)
 	}
 
-	configureEvents()
+	onLedgerUpdatedWorkerPool = workerpool.New(func(task workerpool.Task) {
+		updateIndexer(task.Param(0).(milestone.Index), task.Param(1).(utxo.Outputs), task.Param(2).(utxo.Spents))
+		task.Return(nil)
+	}, workerpool.WorkerCount(workerCount), workerpool.QueueSize(workerQueueSize), workerpool.FlushTasksAtShutdown(true))
+
+	onLedgerUpdated = events.NewClosure(func(index milestone.Index, newOutputs utxo.Outputs, newSpents utxo.Spents) {
+		onLedgerUpdatedWorkerPool.TrySubmit(index, newOutputs, newSpents)
```

Comment:
`TrySubmit` would mean we can loose ledger updates :scream:

Reply:
`workerQueueSize = 10000` not really...

Reply:
If the queue is full, the node should wait, not drop the event!

Reply:
Ok, I'll change it to `Submit`